routing/localchans/manager.go

```diff
@@ -59,6 +63,8 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 
 	haveChanFilter := len(chansToUpdate) != 0
 
+	var errInvalidChans string
```

Comment:
nit: this could also be `[]string` and then we use `strings.Join` rather than trying to smartly append below

Reply:
I'll do that, thank you!